packages/lds/src/index.ts

```diff
@@ -161,10 +164,11 @@ export default async function subscribeToLogicalDecoding(
     } catch (e) {
       console.error("Error during LDS loop:", e.message);
       // Recovery time...
-      loopTimeout = setTimeout(loop, sleepDuration * 10);
-      return;
+      nextSleepDuration *= 10;
```

Comment:
Make this more explicit to avoid refactoring breaking it later

```suggestion
      if (nextSleepDuration >= 0) {
        nextSleepDuration = sleepDuration * 10;
      }
```